code/build.R

```diff
@@ -15,7 +15,7 @@ if(!exists("s")) {
     `PCT schools, Government Target` = "https://npttile.vs.mythic-beasts.com/school/v2/govtarget/{z}/{x}/{y}.png",
     `PCT commuting, Ebikes, ` = "https://npttile.vs.mythic-beasts.com/commute/v2/ebike/{z}/{x}/{y}.png",
     `PCT schools, Go Dutch, ` = "https://npttile.vs.mythic-beasts.com/school/v2/dutch/{z}/{x}/{y}.png",
-    `Existing cycleways (all)` = "https://b.tile-cyclosm.openstreetmap.fr/cyclosm/{z}/{x}/{y}.png",
+    `Existing cycleways and cycle lanes` = "https://b.tile-cyclosm.openstreetmap.fr/cyclosm/{z}/{x}/{y}.png",
```

Comment:
Seems their server has just gone down. Not related to this PR but I may change the basemap to Thunderforest. 

Imagine @mvl22 will approve, right?

Reply:
Can you clarify what you want me to approve?

We have a tilecache you can use - drop me an e-mail with details.

Just to flag that OpenCycleMap is unfortunately likely to be far too busy a layer for this usage - you really need something very neutral IMO.

Reply:
Any suggestions? See here and click on the ` Existing cycleways (all)` check box in the top left hand corner: https://cyipt.github.io/popupCycleways/v0.2/cambridgeshire-and-peterborough/

The current cycleway basemap is also busy.

Reply:
Logically, you aren't going to get a basemap that allows overlays to be easily-visible if that base map is showing detailed cycle infrastructure rather than a very generic map.

There is a technique of applying opacity of e.g. 0.7 to the basemap layer, but that's not a perfect solution.

Reply:
> Logically, you aren't going to get a basemap that allows overlays to be easily-visible if that base map is showing detailed cycle infrastructure rather than a very generic map.

Sounds like a good motivation for a less busy tileset designed for active transport. Know anyone who would be up for considering a `GreyCycleMap` to complement `OpenCycleMap`. Happy to ask the Thunderforest guys, bet there would be demand...